functions/pubsub/main.py

```diff
@@ -62,7 +61,7 @@ def publish(request):
 
 # [START functions_pubsub_subscribe]
 # Triggered from a message on a Cloud Pub/Sub topic.
-def subscribe(event, context):
+def subscribe(event, _ = None):
```

Comment:
This is sample code, so even if we don't use the `context` (or 2nd positional) argument, I would make it clear that this is a context sent from the functions service.

Reply:
OK - so given that, can we put the `None`-handling logic in the test (instead of the sample) to avoid confusing users?

Reply:
What do you mean? Maybe pushing a commit is easier for me to understand?